src/screens/MessageScreen.tsx

```diff
@@ -1,37 +1,112 @@
-import React, { useCallback, useLayoutEffect, useMemo } from 'react';
-import { useHasNewMessagesFromUsers, useStyles, useUser } from '../hooks';
+import React, { useCallback, useLayoutEffect, useMemo, useState } from 'react';
+import { useStyles } from '../hooks';
 import { Avatar, Badge, Divider, List } from 'react-native-paper';
-import { TouchableOpacity } from 'react-native';
+import {
+  NativeScrollEvent,
+  NativeSyntheticEvent,
+  TouchableOpacity,
+} from 'react-native';
 import { ScrollView, View } from 'react-native';
 import { createStyles, useIcons } from '../components/BrandConfigProvider';
 import { HomeStackScreenProps } from '../navigators/types';
 import { t } from 'i18next';
-import { compact, orderBy } from 'lodash';
+import { chunk, compact, orderBy } from 'lodash';
 import { useLookupUsers } from '../hooks/Circles/usePrivatePosts';
 import { ActivityIndicatorView } from '../components';
 import { tID } from '../common';
+import { useUnreadMessages } from '../hooks/useUnreadMessages';
+import { useNotificationManager } from '../hooks/useNotificationManager';
+import { useFocusEffect } from '@react-navigation/native';
 
 export function MessageScreen({
   navigation,
   route,
 }: HomeStackScreenProps<'Home/Messages'>) {
   const { recipientsUserIds } = route.params;
```

Comment:
We can move this into a hook just get the `messageTile`'s `id` from params.  (In a future PR)

Reply:
👍